content/metadata-stats/metadata/subscribe.textile

```diff
@@ -93,11 +93,36 @@ await channel.subscribe('channel.closed', event_listener)
 
 h2(#log). Log events
 
-The @[meta]log@ and @[meta]log:push@ metachannels publish events that aren't otherwise available to clients. For example, if Ably receives an error response when delivering a webhook to your endpoint, Ably sends an event to @[meta]log@ with information about the error.
+The @[meta]log@ and "@[meta]log:push@":/push#Error metachannels publish events that aren't otherwise available to clients.
 
 Errors where the client can be directly notified are **not** published to the metachannels. For example, if a client attempts to publish a message but exceeds a channel rate limit, the client will be notified by an error callback passed to the "publish()":/api/realtime-sdk/channels#publish method.
 
-@[meta]log@ publishes errors related to all resources other than Push Notifications, which are published to @[meta]log:push@.
+The following example subscribes to the @[meta]log@ channel:
+
+```[javascript]
+const channel = ably.channels.get('[meta]log');
+```
+
+An example scenario for receiving error logs: If Ably encounters an error when delivering a webhook to your endpoint, it sends an event to @[meta]log@ with details about the error.
+
+The following example shows the @[meta]log@ error logs for a "webhook":/general/webhooks delivery failure:
+
+```[json]
```

Comment:
This isn't valid JSON - you need double quotes and to wrap it in { } for consistency. I'm not sure if the _ is intentional or not too?

Reply:
This is still outstanding.